frontend-react/src/components/shared/Table/Table.tsx

```diff
@@ -0,0 +1,225 @@
+import classnames from "classnames";
+import sanitizeHtml from "sanitize-html";
+import { useMemo, useState } from "react";
+import { Icon } from "@trussworks/react-uswds";
+
+import { convertStringToSortableRepresentation } from "../../../utils/misc";
+
+import styles from "./Table.module.scss";
+
+const FilterOptionsEnum = {
```

Comment:
Can we not use an actual `enum` here?  I think the `Enum` suffix is redundant in any case.